misc/lint.sh

```diff
@@ -42,7 +42,8 @@ lint_html () {
     'Illegal character in query: "\|" is not allowed' \
     '"(autocorrect|autocapitalize)" not allowed on element "input"' \
     '"align" attribute on the "(td|th)" element is obsolete' \
-    'Document uses.* Unicode Private Use Area';
+    'Document uses.* Unicode Private Use Area' \
+    'Attribute "autocomplete" is only allowed when the input type is "color", "date", "datetime-local", "email", "hidden", "month", "number", "password", "range", "search", "tel", "text", "time", "url", or "week".';
```

Comment:
The corresponding fix has been merged, so this one is unnecessary.

Reply:
Removed